frontend/src/components/desktop_apps/screens/TermialAppScreen.tsx

```diff
@@ -21,13 +21,22 @@ export default function TerminalAppScreen() {
     function generateResponse(takenInput: string) {
```

Comment:
What is up with this entire lambda? `inputValue` is returned before `generateResponse()` is declared, which I assume is being hoisted? If `generateResponse` is only being once it might as well be deleted and have it's logic just be part of `checkForCommands`.